create_config.py

```diff
@@ -1,21 +1,24 @@
-from jinja2 import Template
+import os
 import json
+from jinja2 import Template
 
+DEBUG = True
+output_dir = "output_configs"
```

Comment:
The configuration files have to be in a specific folder, I can't create a new one.

Reply:
Ok, I will change this then. Is it the same folder of the clients json file?

If you are ok with that, I would add an optional cmd line argument. The default value would be the same folder as the script, but if you specify a folder it will copy the output there. (It is handy for debugging, so we don't create necessarily a lot of files in the code folder)